CPT331.Core/ObjectModel/ReadOnlyDataObject.cs

```diff
@@ -0,0 +1,131 @@
+﻿#region Using References
+
+using System;
+
+#endregion
+
+namespace CPT331.Core.ObjectModel
+{
+	/// <summary>
+	/// Represents a ReadOnlyDataObject type, which contains immutable information obtained fron a data source.
+	/// </summary>
+	public class ReadOnlyDataObject
```

Comment:
This is the base object model type (atm anyway). Allows for many object model types to be treated the same way.